engine/src/hndlrlst.cpp

```diff
@@ -710,6 +710,38 @@ bool MCHandlerlist::enumerate(MCExecContext& ctxt, bool p_first, uindex_t& r_cou
 	return p_first;
 }
 
+bool MCHandlerlist::listconstants(MCHandlerlistListConstantsCallback p_callback, void *p_context)
+{
+	for(uint2 i = 0; i < nconstants; i++)
+	{
+		if (!p_callback(p_context,
+						&cinfo[i]))
+		{
+			return false;
+		}
+	}
+	
+	return true;
+}
+
+bool MCHandlerlist::listhandlers(MCHandlerlistListHandlersCallback p_callback, void *p_context)
+{
+	for(int t_type = 0; t_type < HT_MAX - 1; t_type++)
```

Comment:
Would this be better as `for(int t_type = HT_UNDEFINED + 1; type < HT_MAX;...)` to avoid the `t_type + 1` later? As it is you have to look up the `Handler_type` enum to see why you've done it